src/foam/nanos/crunch/UserCapabilityJunctionRefine.js

```diff
@@ -67,6 +75,85 @@ foam.CLASS({
       Set when the ucj is first granted.
       `
     }
+  ],
+
+  methods: [
+
+    {
+      name: 'saveDataToDAO',
+      args: [
+        {
+          name: 'x',
+          type: 'Context'
+        },
+        {
+          name: 'capability',
+          type: 'Capability'
+        },
+        {
+          name: 'putObject',
+          type: 'Boolean'
+        }
+      ],
+      javaType: 'foam.core.FObject',
+      documentation: `
+        We may or may not want to store the data in its own dao, based on the nature of the data.
```

Comment:
So as long as the data model consists of properties from one model (i.e User) then this should be fine. Are we ever going to have a case where the data model consists of properties from multiple models (i.e. User and Account or smth)? @Nauna @noodlemoodle

Reply:
No, the idea is to have minimal models to facilitate reuse. Therefore can't ever see something with 2 models and even if that were the case then I feel like, you would be correct in creating a 3rd model that combined them- thus back to one model.